code/Control/UserDefinedFormController.php

```diff
@@ -672,4 +678,20 @@ function (){
 
         return $result;
     }
+
+    /**
+     * Check validity of email and return array of valid emails
+     */
+    protected function validEmailsToArray(?string $emails): array
```

Comment:
Could this even be `private`?

Reply:
I'm not sure if anyone extends `UserDefinedFormController` class, but I would still like to leave access to this method in child classes. Also, this method does not work with any object or class properties and does not set new values.
But if you think that the best practice is to use `private`, I will be happy to make these changes.

Reply:
Please default to `private` unless there's a compelling reason to use `protected`

Reply:
DONE